react-state3/react-state-3_journal-app-form/src/components/EntryForm/index.js

```diff
@@ -1,18 +1,57 @@
 import "./EntryForm.css";
 import Button from "../Button";
+import { useState } from "react";
+
+export default function EntryForm({ onAddEntry }) {
+  const [formData, setFormData] = useState({ motto: "", notes: "" });
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    // Create a new entry object with the values from the form
+    const newEntry = {
+      motto: formData.motto,
+      notes: formData.notes,
+    };
+
+    // Call the onAddEntry function and pass the new entry
+    onAddEntry(newEntry);
+
+    // Reset the form
+    setFormData({ motto: "", notes: "" });
```

Comment:
creative solution, alternatively good solution Jan showed us in class: 
`event.target.reset();`